src/dbSta/src/CMakeLists.txt

```diff
@@ -85,6 +85,7 @@ swig_lib(NAME          dbSta
 
 target_sources(dbSta
   PRIVATE
+    MakeDbSta.cc
     dbSta.cc
```

Comment:
If dbSta.cc is in in dbSta and not dbSta_lib then its header dbSta.hh shouldn't be public.

Reply:
I think we should be able to move `dbSta.cc` to `dbSta_lib` once we've abstracted the GUI dependency (and somehow addressed the OpenRoad observer). Those two things still couple it to the OpenRoad object after this change. This is an intermediate step that we felt was pretty self-contained and smaller / easier to review vs needing to "go for broke" of doing the full separation shown here plus gui plus openroad observer.

I think as a separate issue from whether `dbSta.cc` belongs in dbSta_lib (which would imply we could unit test it), `dbSta.hh` is used by a bunch of other components (which directly call methods on the dbSta object), so I think it would have to continue to be public for those purposes?

LMK if that makes sense or if I misunderstood.

Reply:
It's helpful to let me know that you view this as an intermediate step.  If you plan to continue on and complete the separation then I'm ok with this as just forward progress.

Reply:
I'll put the additional context in the PR next time of the succession of expected followup changes. Note, though, I was hoping this was a self contained change that made sense on its own even though it is towards the broader goal -- LMK if you don't feel like that's the case since that'd mean to me that some more discussion would be warranted.